src/base/numbers.c

```diff
@@ -74,3 +74,42 @@ int _getHexString(char *buffer, unsigned long hexnumber, bool use_capital) {
     }
     return size;
 }
+
+// Unsigned
+int itoa(char *buffer, unsigned long number, int base, bool use_capital) {
+	if (base < 1 || base > 36) return 0;
+	char *pointer, *pointerbase;
```

Comment:
I'm not sure how @dreamos82 feels about this, declaring multiple variables in one line?

Reply:
That is the way it is done in _getUnsignedDecString() right before it. I'm just copying the formatting from there.

Reply:
I spoke to dreamos82 on discord and he didn't mind so feel free to ignore this one :)

Reply:
as i said to @DeanoBurrito  i don't have strong feelings about it, i tend to prefer them being split, but sometime i don't do it as you can see. But yeah i think is more readable having separate variable per lines.